Software/Source/BrstPktCnt.py

```diff
@@ -0,0 +1,50 @@
+import json
+#Global Variable
+a=0
+b=0
+c=0
+d=0
+e=0
+#count burst wise packet count
+def BrstPktCnt(packet):
+    if(packet=='0'):
+        global a
+        a=a+1
+    elif(packet=='1'):
+        global b
+        b=b+1
+    elif(packet=='2'):
+        global c
+        c=c+1
+    elif(packet=='3'):
+        global d
+        d=d+1
+    elif(packet=='4'):
+        global e

```

Comment:
Declare all variables in one place. 
Refer general structure/body of a code.

Review all your deliverables and incorporate these changes..

Summary:
1. Meaningful names for variables, functions, etc
2. Follow standard coding practices to develop a properly structured code.